packages/playground/website/src/lib/site-metadata.ts

```diff
@@ -89,7 +89,7 @@ export async function createSiteMetadata(
 		blueprintSource = resolvedBlueprint.source;
 	}
 
-	const compiledBlueprint = await compileBlueprint(blueprint);
```

Comment:
this breaks the bulk of e2e tests, let's figure avoid calling it and using the results for Blueprints v2